homeassistant/helpers/issue_registry.py

```diff
@@ -181,6 +186,7 @@ def async_delete(self, domain: str, issue_id: str) -> None:
             return
 
         self.async_schedule_save()
+        # Replace by Callback instead?...
```

Comment:
Please don't add comments, do a self-review instead where you add questions or remarks

Reply:
Alright, removing that.

It's an open question based on an earlier PR thread relating event firing: https://github.com/home-assistant/core/pull/94393#issuecomment-1594720799, where @bdraco mentions the cost of firing events rather than using specifically registered callbacks.

I'm comparing the `issue_registry` here to `persistent_notifications` that use callback registration instead of these event firings and it seems to be something the issues component should use as well. Though admittedly that should be a separate PR.

Reply:
Issues are expected to be much more rare so firing an event shouldn't be a problem